tokio/src/sync/broadcast.rs

```diff
@@ -627,6 +616,24 @@ impl<T> Sender<T> {
     }
 }
 
+impl<T> Shared<T> {
+    fn new_receiver(self: Arc<Self>) -> Receiver<T> {
```

Comment:
This is going to have to be `new_receiver(arc: Arc<Self>)` instead since the loom arc cannot be used as a self-parameter.

Reply:
I was wondering if that would be an issue, I didn't realize loom had it's own arc though